pyproject.toml

```diff
@@ -13,8 +13,11 @@ repository = "https://github.com/strawberry-graphql/strawberry"
 documentation = "https://strawberry.rocks/"
 
 classifiers = [
+    "Development Status :: 5 - Production/Stable",
```

Comment:
@jkimbo @BryceBeagle happy to add this? :D

Reply:
Yep I think so!